pkg/mcs/tso/server/server.go

```diff
@@ -136,10 +143,10 @@ func (s *Server) Close() {
 		return
 	}
 
-	log.Info("closing server")
-
-	s.grpcServer.Stop()
-	s.httpServer.Shutdown(s.ctx)
+	log.Info("Closing TSO Server ...")
```

Comment:
It is better to use lowercase letters for consistency with other logs.

Reply:
You're right. Will do.

Reply:
Also changed in the other places where the style has the same problem.